config/owasp/suppressions.xml

```diff
@@ -32,4 +32,11 @@
         <packageUrl regex="true">^pkg:maven/org\.glassfish/jakarta\.el@.*$</packageUrl>
         <cve>CVE-2021-28170</cve>
     </suppress>
-</suppressions>
\ No newline at end of file
+    <suppress until="2021-11-16">
+        <notes><![CDATA[
+   file name: lang-tag-1.4.4.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.nimbusds/lang\-tag@.*$</packageUrl>
```

Comment:
we have already added the suppression for this jar. we just need to add this cve alone - like this https://github.com/hmcts/rd-judicial-api/pull/162/files